ups/lsst_distrib.table

```diff
@@ -15,3 +15,5 @@ setupRequired(display_matplotlib)
 setupRequired(cp_pipe)
 setupRequired(validate_drp)
 setupRequired(meas_mosaic)
+setupRequired(fgcm)
```

Comment:
Do not add fgcm. The dependency you are adding to lsst_distrib is fgcmcal and that will itself bring in fgcm.